torch/_dynamo/variables/dicts.py

```diff
@@ -196,37 +277,7 @@ def modifed(self, items, **options):
         return self.clone(items=items, **options)
 
     def unpack_var_sequence(self, tx):
-        options = VariableTracker.propagate([self])
-        val = self.items
-        result = [ConstDictVariable._key_to_var(tx, k, **options) for k in val.keys()]
-        return result
-
-    @classmethod
-    def get_key(cls, arg: VariableTracker):
-        if isinstance(arg, TensorVariable) and arg.specialized_value is not None:
-            return arg.specialized_value
-        else:
-            return arg.as_python_constant()
-
-    @classmethod
-    def is_valid_key(cls, key):
-        return (
-            key.is_python_constant()
-            or isinstance(key, TensorVariable)
-            and key.specialized_value is not None
-            or isinstance(key, ConstantVariable)
-            and key.python_type() is torch.dtype
-        )
-
-    @classmethod
-    def _key_to_var(cls, tx, key, **options):
-        from .builder import VariableBuilder
-
-        if istensor(key):
-            return VariableBuilder(tx, GlobalWeakRefSource(global_key_name(key)))(key)
-        else:
-            assert ConstantVariable.is_literal(key)
-            return ConstantVariable.create(key, **options)
+        return [x.vt for x in self.items.keys()]
```

Comment:
Should we propagate the guards here? I do not see why should we a priori. If a dictionary becomes invalid that shouldn't invalidate its previous values?

Reply:
This won't matter once https://github.com/pytorch/pytorch/pull/111415 lands